build_tools/scripts/byo_llvm.sh

```diff
@@ -69,7 +69,7 @@ do_build_llvm() {
   echo "*********************** BUILDING LLVM *********************************"
   main_build_dir="${LLVM_BUILD_DIR}/llvm"
   main_install_dir="${LLVM_INSTALL_DIR}/llvm"
-  targets_to_build="${LLVM_TARGETS_TO_BUILD:-X86}"
+  targets_to_build="${LLVM_TARGETS_TO_BUILD:-X86;ARM;AArch64;RISCV}"
```

Comment:
OK. I realize now, that the need for this is only because of a "bug" in the ukernels bitcode setup: it only checks whether `IREE_TARGET_BACKEND_LLVM_CPU` and then it tries to build the bitcode for ALL architectures. It needs to act more precisely based on the enabled LLVM architecture backends. So I'll do your suggestion here now and separately I'll send a PR to fix that.